### PR TITLE
FOGL-5592: Update existing system test with respect to FOGL-5573 changes and add new system test for OMF naming scheme (FOGL-5573)

### PR DESCRIPTION
**Pull request by YashTatkondawar**

### CODE REVIEW

**Review by praveen-garg — DISMISSED** [review]
1) start_north_pi_server_c_web_api  can be aliased as start_north_task_omf_web_api 
2)  type_id = 1 fixed? when it does change?
3) re-config okay? or cleanup required before that.
4) Re-confirm, verification we see here is valid
   ```
      Concise
          recorded_datapoint = "{}".format(south_asset_name)
          # Name of asset in the PI server
          pi_asset_name = "{}".format(south_asset_name)
      Use type suffix        
          recorded_datapoint = "{}".format(south_asset_name)
          # Name of asset in the PI server
          pi_asset_name = "{}-type{}".format(south_asset_name, type_id)
      Use Attribute Hash
          type_id = 1
          recorded_datapoint = "{}measurement_{}".format(type_id, south_asset_name)
          # Name of asset in the PI server
          pi_asset_name = "{}".format(south_asset_name)
      Backward compatibility
          type_id = 1
          recorded_datapoint = "{}measurement_{}".format(type_id, south_asset_name)
          # Name of asset in the PI server
          pi_asset_name = "{}-type{}".format(south_asset_name, type_id)
     ```

**Comment by YashTatkondawar** [comment]
> 1. start_north_pi_server_c_web_api  can be aliased as start_north_task_omf_web_api

Done

> 2. type_id = 1 fixed? when it does change?

Whenever a new property is added to an asset, a new type is created. In this test, we are sending the same type of data so type_id will always be 1.

> 3. re-config okay? or cleanup required before that.

Filed FOGL-5605 for the issue in re-config.

> 4. Re-confirm, verification we see here is valid
>    ```
>       Concise
>           recorded_datapoint = "{}".format(south_asset_name)
>           # Name of asset in the PI server
>           pi_asset_name = "{}".format(south_asset_name)
>       Use type suffix        
>           recorded_datapoint = "{}".format(south_asset_name)
>           # Name of asset in the PI server
>           pi_asset_name = "{}-type{}".format(south_asset_name, type_id)
>       Use Attribute Hash
>           type_id = 1
>           recorded_datapoint = "{}measurement_{}".format(type_id, south_asset_name)
>           # Name of asset in the PI server
>           pi_asset_name = "{}".format(south_asset_name)
>       Backward compatibility
>           type_id = 1
>           recorded_datapoint = "{}measurement_{}".format(type_id, south_asset_name)
>           # Name of asset in the PI server
>           pi_asset_name = "{}-type{}".format(south_asset_name, type_id)
>    ```
lgtm.

File: tests/system/python/conftest.py [hunk 1]
```diff
@@ -178,9 +179,10 @@ def _start_north_pi_server_c(fledge_url, pi_host, pi_port, pi_token, north_plugi
 
 @pytest.fixture
 def start_north_pi_v2_web_api():
```
Comment (praveen-garg) [inline tests/system/python/conftest.py:181]: we can also make it `start_north_task_omf_web_api`

File: tests/system/python/conftest.py [hunk 2]
```diff
@@ -245,7 +250,7 @@ def _start_north_omf_as_a_service(fledge_url, pi_host, pi_port, pi_db="Dianomic"
 
 
 start_north_pi_server_c = start_north_pi_v2
-start_north_pi_server_c_web_api = start_north_pi_v2_web_api
```
Comment (praveen-garg) [inline tests/system/python/conftest.py:248]: I would suggest to keep 

`# As we had name pi_server, pi_v2 in old versions for OMF PI server things
start_north_pi_server_c_web_api = start_north_pi_v2_web_api = start_north_task_omf_web_api

**Review by praveen-garg — APPROVED** [review]

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**